### DISCUSSION TITLE
After the update, video download speeds have become very slow.

### DISCUSSION THREAD

**Discussion by Tanggue** [aclap-dev/video-downloadhelper#1113] [Help and Q&A]
### Your question

Thank you for your good service.
After the recent update, my download speed has become very slow (130kb/s).
It takes 4 hours to download a 2-hour video.
Is there a solution to this problem?

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: ko
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "used_history_button": true,
    "download_directory": "D:\\",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\kangt\\Downloads"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 13,
    "successfull_dl": 70,
    "record_download_history": true
}

**Comment by Mattmjs** [comment]
Most likely it's the site throttling the speed.
If you want to see if others have the same problem then you share a link to a video.
